<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QWidget" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>460</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Graph calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius:10px;
background-color: rgb(66, 66, 66)</string>
  </property>
  <widget class="QLabel" name="xmax">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>X max</string>
   </property>
  </widget>
  <widget class="QLabel" name="xmin">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>X min</string>
   </property>
  </widget>
  <widget class="QLabel" name="ymax">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>370</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Y max</string>
   </property>
  </widget>
  <widget class="QLabel" name="ymin">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Y min</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="max_edit_x">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="min_edit_x">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>-10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="max_edit_y">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="min_edit_y">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>-5</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>280</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="plot_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>320</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
background-color: rgb(98, 160, 234);
color: rgb(0, 0, 0);
}
QPushButton:hover {
    background-color : rgb(44, 96, 137);
}
QPushButton:pressed {
    background-color : rgb(33, 33, 33);
}


</string>
   </property>
   <property name="text">
    <string>Отрисовать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stock_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
background-color: rgb(98, 160, 234);
color: rgb(0, 0, 0);
}
QPushButton:hover {
    background-color : rgb(44, 96, 137);
}
QPushButton:pressed {
    background-color : rgb(33, 33, 33);
}


</string>
   </property>
   <property name="text">
    <string>Стандартные значения </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="label_funk">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>620</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Место для ввода функции&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Ваша функция</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius:10px;
background-color: rgb(98, 160, 234);
color: rgb(0, 0, 0);
}
QPushButton:hover {
    background-color : rgb(44, 96, 137);
}
QPushButton:pressed {
    background-color : rgb(33, 33, 33);
}


</string>
   </property>
   <property name="text">
    <string>Отчистить</string>
   </property>
  </widget>
  <zorder>plot_Button</zorder>
  <zorder>xmax</zorder>
  <zorder>xmin</zorder>
  <zorder>ymax</zorder>
  <zorder>ymin</zorder>
  <zorder>max_edit_x</zorder>
  <zorder>min_edit_x</zorder>
  <zorder>max_edit_y</zorder>
  <zorder>min_edit_y</zorder>
  <zorder>widget</zorder>
  <zorder>stock_Button</zorder>
  <zorder>label_funk</zorder>
  <zorder>clear_Button</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
